pkg/agent/config/node_config.go

```diff
@@ -59,6 +59,23 @@ var (
 	VirtualServiceIPv6 = net.ParseIP("fc01::aabb:ccdd:eeff")
 )
 
+type NodeType uint8
+
+const (
+	K8sNode NodeType = iota
+	ExternalNode
+)
+
+func (t NodeType) String() string {
+	switch t {
+	case K8sNode:
+		return "K8sNode"
+	case ExternalNode:
+		return "ExternalNode"
+	}
+	return "unknown"
```

Comment:
Unify the case of K8sNode, ExternalNode, **unknown**. I think all current configuration value use lower camel case

Reply:
updated.